### PR TITLE
Left Side

### PR DESCRIPTION
**Pull request by agmass**
Some basic left side stuff. Healthbar, arrow positions are not done miss anims still play on bf, If anyone wants to finish them go ahead.

It is in the options menu but i didn't add it to the save since i feel like it would be weird as hell to load up fnf and randomly start playing left side because you forgot to turn it off.

### CODE REVIEW

**Comment by MaxlionButMyNameIsTakenSoThisIsmyName** [comment]
ah also, i changed pause and gameover since stuff wouldn't load right if you pressed "Restart Song" or pressed enter on gameover.

**Comment by frantastic24** [comment]
Isn't this the same as #3549

**Comment by Gui-iago** [comment]
I'd recommend marking this as a draft if you're not finished with it yet

**Comment by Gui-iago** [comment]
> Isn't this the same as #3549

it has some different code for it, so it's not an exact duplicate, let the man make his code and at the end we will see what it's all about

**Comment by frantastic24** [comment]
I'm not against them doing their own version of it but as you just said this is incomplete, on top of doing (or supposed to be doing) the exact same thing as another PR that is already fully working

**Comment by MaxlionButMyNameIsTakenSoThisIsmyName** [comment]
> Isn't this the same as #3549

That one's code was a little confusing to me, I just thought inverting the numbers would a easier way to do it. Also uses ClientPrefs and Options instead of gameplay changers.

**Comment by MaxlionButMyNameIsTakenSoThisIsmyName** [comment]
Also i tried dividing/multiplying the numbers but 0 * 2 doesn't make 4 and 4 % 2 doesn't make 0 lol

**Comment by frantastic24** [comment]
I'd argue it's better as a gameplay changer (easier to access and faster)
> 4 % 2 doesn't make 0
Yes it does

**Comment by MaxlionButMyNameIsTakenSoThisIsmyName** [comment]
> > 4 % 2 doesn't make 0
> Yes it does

O I thought it ment divided then rounded, I’m dumb

**Comment by Gui-iago** [comment]
@MaxlionButMyNameIsTakenSoThisIsmyName is this still work in progress?

**Comment by MaxlionButMyNameIsTakenSoThisIsmyName** [comment]
> I'd argue it's better as a gameplay changer (easier to access and faster)
> 
> > 4 % 2 doesn't make 0
> 
> Yes it does



> @MaxlionButMyNameIsTakenSoThisIsmyName is this still work in progress?

Chart flipping and anims  works, arrows, healthbar, misses don’t

**Comment by MaxlionButMyNameIsTakenSoThisIsmyName** [comment]
Adding those right now.

**Comment by Steviegt6** [comment]
> > > 4 % 2 doesn't make 0
> > > Yes it does
> 
> O I thought it ment divided then rounded, I’m dumb

The module (%) operator returns the *remainder* of a division operating. 4/2 = 2 r0, so there is a remainder of 0. 3/2 = 1 r1, so there is a remainder of 1, etc.

**Comment by MaxlionButMyNameIsTakenSoThisIsmyName** [comment]
> > > > 4 % 2 doesn't make 0
> > > > Yes it does
> > 
> > 
> > O I thought it ment divided then rounded, I’m dumb
> 
> The modulo (%) operator returns the _remainder_ of a division operation. 4/2 = 2 r0, so there is a remainder of 0. 3/2 = 1 r1, so there is a remainder of 1, etc.

Googled it before you told me >8)

File: source/options/GameplaySettingsSubState.hx [hunk 1]
```diff
@@ -56,6 +56,13 @@ class GameplaySettingsSubState extends BaseOptionsMenu
 			false);
 		addOption(option);
 
+		var option:Option = new Option('Left Side',
+		'If checked, You play the opposite side. Does not change healthbar or arrow location.',
```
Comment (HiroMizuki) [inline source/options/GameplaySettingsSubState.hx:60]: not a big error, but, in the rules, uppercases are not used after a comma, only after a full stop punctuation
sorry if i'm disturbing here

**Review by HiroMizuki — APPROVED** [review]
i think i'll approve it lmao

**Pull request closed without merging**